src/framework/audio/internal/audioconfiguration.h

```diff
@@ -72,6 +75,8 @@ class AudioConfiguration : public IAudioConfiguration
     io::path_t knownAudioPluginsFilePath() const override;
 
 private:
+    int m_audioDelayCompensate;
```

Comment:
For "stupid" types like integers, floats, bools, pointers, it's better to add a default value (even just `= 0` is better than nothing), to prevent indeterministic behaviour (which is difficult to debug).